multiuser/permission/che-multiuser-permission-workspace/src/test/java/org/eclipse/che/multiuser/permission/workspace/server/jpa/WorkspaceTckModule.java

```diff
@@ -100,6 +101,7 @@ protected void configure() {
                 org.eclipse.che.api.workspace.server.model.impl.devfile.VolumeImpl.class)
             .addEntityClass(
                 "org.eclipse.che.api.workspace.server.model.impl.ProjectConfigImpl$Attribute")
+            .addEntityClass(SerializableConverter.class)
```

Comment:
I know that you just use existing SPI, but `SerializableConverter` is not really entity and it looks a bit confusing.
Maybe it makes sense to add new method to `PersistTestModuleBuilder` that would do the same as `addEntityClass` but would be named as `addClass`?

Reply:
Done